internal/utils/jobhelper.go

```diff
@@ -16,14 +17,17 @@ import (
 type Status string
 
 const (
-	jobHashAnnotation = "kmm.node.kubernetes.io/last-hash"
+	BuildType = "build"
+	SignType  = "sign"
```

Comment:
`utils.BuildType` is not very explicit IMHO.
```suggestion
	JobTypeBuild = "build"
	JobTypeSign  = "sign"
```
Also wondering if the Job helper should have its own package.

Reply:
yes, it can go both ways, but eventually i thought that its main purpose is to assist other package with limited functionalities, not as a standalone package, that's why i asked Chris to put it in utils. Both jobhelper and volume helper contain command functions for build and sign packages, that is additional reason why  it seems to me more appropriate to put them in utils

Reply:
fixed the rename of types